scalameter-core/src/main/scala/org/scalameter/Measurer.scala

```diff
@@ -12,6 +17,12 @@ trait Measurer extends Serializable {
   def name: String
   def measure[T, U](context: Context, measurements: Int, setup: T => Any, tear: T => Any, regen: () => T, snippet: T => Any): Seq[Double]
   def units: String
+
+  /** Prepares fresh context that should be concatenated with the setup context. */
```

Comment:
Could you please improve this comment:
- which context is this method called on?
- why is the method useful?
- how does the method change the context?
- when is this method called?
- who calls the method?